core/es_isolate.rs

```diff
@@ -80,25 +82,10 @@ impl DerefMut for EsIsolate {
   }
 }
 
+// TODO(ry): a V8 Isolate cannot actually be moved between threads without the
+// use of a Locker, therefore EsIsolate should not implement the `Send` trait.
 unsafe impl Send for EsIsolate {}
 
-impl Drop for EsIsolate {
-  fn drop(&mut self) {
-    let isolate = self.core_isolate.v8_isolate.as_ref().unwrap();
-    // Clear persistent handles we own.
-    {
-      let mut locker = v8::Locker::new(&isolate);
-      let scope = locker.enter();
-      for module in self.modules.info.values_mut() {
-        module.handle.reset(scope);
-      }
-      for handle in self.dyn_import_map.values_mut() {
-        handle.reset(scope);
-      }
-    }
-  }
-}
```

Comment:
\o/

Reply:
You realize that this section costs you $1.25 per line removed, right? 😈

Reply:
Indeed. I owe you $20.